doc/migrations/3.3.0_3.3.0plus.md

```diff
@@ -0,0 +1,58 @@
+## New field in Message Archive Management MUC entries: Sender ID
+
+As a part of ensuring GDPR compliance, it turned out it's essential to be able to efficiently query MAM MUC data via sender ID (to retrieve user's personal data).
+Originally, the sender JID could be found only as a part of encoded XML message element, so finding all items sent by a certain user would be **extremely** inefficient (or rather: anti-efficient).
+MongooseIM 3.3.0++ uses modified schema for MAM MUC backends which enables way more efficient extraction.
+
+You may find migration instructions specific to your MAM backend below.
+
+### RDBMS
+
+#### Step 1
+
+Please execute following SQL statements on your MIM database:
+
+**MySQL**
+
+```
+ALTER TABLE mam_muc_message ADD COLUMN sender_id INT UNSIGNED;
+CREATE INDEX i_mam_muc_message_sender_id USING BTREE ON mam_muc_message(sender_id);
+```
+
+**PostgreSQL**
+
+```
+ALTER TABLE mam_muc_message ADD COLUMN sender_id INT;
+CREATE INDEX i_mam_muc_message_sender_id ON mam_muc_message USING BTREE (sender_id);
+```
+
+**MSSQL**
+
+```
+ALTER TABLE [dbo].[mam_muc_message] ADD sender_id bigint;
+CREATE INDEX i_mam_muc_message_sender_id ON mam_muc_message(sender_id);
+```
+
+#### Step 2
+
+Now you have a schema that is compatible with MIM 3.3.0++ but isn't GDPR-compliant yet because the new column has no meaningful data.
+
+Please pick your favourite scripting/programming language and populate the new column with [dedicated script's help](jid-from-mam-muc-script.md) help.
```

Comment:
Please pick your favourite scripting/programming language and populate the new column with the help of a [dedicated script](jid-from-mam-muc-script.md).